packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,48 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
+import {randomBytes} from 'crypto';
+import * as appInsights from 'applicationinsights';
+
+appInsights.setup('795006ca-cf54-40ee-8bc6-03deb91401c3');
+const telClient = appInsights.defaultClient;
+if (!telClient.commonProperties.sessionId) {
+  telClient.commonProperties.sessionId = randomBytes(16).toString('hex');
+  function sanitizeStackTrace(envelope: any /*context: any*/): boolean {
+    if (envelope.data.baseType === 'ExceptionData') {
+      const data = envelope.data.baseData;
+      if (data.exceptions && data.exceptions.length > 0) {
+        for (let i = 0; i < data.exceptions.length; i++) {
+          const exception = data.exceptions[i];
+          for (const frame of exception.parsedStack) {
+            const parens = frame.method.indexOf('(');
```

Comment:
not sure what you meant here since I am using the index of the parens to extract the method name

---
In reply to: [516241037](https://github.com/microsoft/react-native-windows/pull/6403#discussion_r516241037) [](ancestors = 516241037)